File: leetcode/js/find-indices-of-stable-mountains.js
/**
 * @param {number[]} height
 * @param {number} threshold
 * @return {number[]}
 */
var stableMountains = function (height, threshold) {
  let result = [];

  for (let i = 1; i < height.length; i++) {
    if (height[i - 1] > threshold) {
      result.push(i);
    }
  }

  return result;
};

File: leetcode/rust/find-indices-of-stable-mountains.rs
impl Solution {
    pub fn stable_mountains(height: Vec<i32>, threshold: i32) -> Vec<i32> {
        let mut result = Vec::new();

        for i in 1..height.len() {
            if height[i - 1] > threshold {
                result.push(i as i32);
            }
        }

        result
    }
}
